src/main/java/org/sagebionetworks/web/client/widget/entity/annotation/AnnotationEditorV2.java

```diff
@@ -1,23 +0,0 @@
-package org.sagebionetworks.web.client.widget.entity.annotation;
-
-import com.google.gwt.user.client.ui.Widget;
-import com.google.inject.Inject;
-
-public class AnnotationEditorV2 implements AnnotationEditorV2View.Presenter {
```

Comment:
This widget showed the new editor in its own modal in alpha mode. Since the editor is now contained in the EntityModal React component, this is no longer used.